src/crypto/crypto_mbedtls-ec.c

```diff
@@ -41,7 +43,19 @@ int crypto_rng_wrapper(void *ctx, unsigned char *buf, size_t len)
 
 const struct crypto_bignum *crypto_ec_get_a(struct crypto_ec *e)
 {
-	return (const struct crypto_bignum *)&e->group.A;
+	int ret;
```

Comment:
ret doesn't seem to be getting used?

Reply:
It's used by the macro `MBEDTLS_MPI_CHK` this needs both `ret` and `cleanup` to be defined.